app/src/main/java/com/bunbeauty/papakarlo/feature/productdetails/ProductDetailsFragment.kt

```diff
@@ -142,38 +169,175 @@ class ProductDetailsFragment :
 
                 is ProductDetailsUi.Error -> {
                     ErrorScreen(mainTextId = R.string.common_error) {
-                        onAction(ProductDetailsState.Action.Init(menuProductUuid))
+                        onAction(
+                            ProductDetailsState.Action.Init(
+                                menuProductUuid = menuProductUuid,
+                                selectedAdditionUuidList = additionUuidList
+                            )
+                        )
                     }
                 }
             }
         }
     }
 
     @Composable
-    private fun ProductDetailsSuccessScreen(menuProductUi: ProductDetailsUi.Success.MenuProductUi?) {
+    private fun ProductDetailsSuccessScreen(
+        menuProductUi: ProductDetailsUi.Success.MenuProductUi?,
+        onAction: (ProductDetailsState.Action) -> Unit
+    ) {
         menuProductUi?.let {
-            Column(
+            LazyColumn(
                 modifier = Modifier
                     .fillMaxWidth()
-                    .verticalScroll(rememberScrollState())
             ) {
-                ProductCard(menuProductUi)
-                Spacer(modifier = Modifier.height(FoodDeliveryTheme.dimensions.scrollScreenBottomSpace))
+                item {
+                    ProductCard(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(horizontal = FoodDeliveryTheme.dimensions.mediumSpace)
+                            .padding(top = FoodDeliveryTheme.dimensions.mediumSpace),
```

Comment:
dp

Reply:
do u mean than i need to set value like 16.dp instead of FoodDeliveryTheme.dimensions.mediumSpace